staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -107,15 +110,25 @@ type DiffOptions struct {
 	FieldManager    string
 	ForceConflicts  bool
 
-	Selector         string
```

Comment:
This object already had too many fields, I think it's way too many now, can we refactor this instead?

Reply:
Good idea, I'll reorganize structs in here.